packages/manager/modules/support/src/tickets/new-ticket/new-ticket.html

```diff
@@ -52,6 +52,9 @@
        data-translate-values="{ state: 'support.tickets.ticket({ id: $ctrl.ticketId })' }"
        data-translate-compile>
     </p>
+
+    <support-tickets-new-feedback
+        data-language="$ctrl.TranslateService.getUserLocale(true)"></support-tickets-new-feedback>
```

Comment:
As you always get the `language` the same way, any reason why you don't do it in the component itself ?

Reply:
It don't want it linked to the API, components should be as dumb as a rock 😄